CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default assignment
+* @turbocat2001 @rgimad
```

Comment:
Please specify @mkostoevr as owner of:
- [kernel/test](https://github.com/kolibrios-nextgen/kolibrios-ng/tree/main/kernel/test)
- [kernel/runtests.py](https://github.com/kolibrios-nextgen/kolibrios-ng/tree/main/kernel/runtests.py)
- [kernel/asmxygen.py](https://github.com/kolibrios-nextgen/kolibrios-ng/tree/main/kernel/asmxygen.py)
- [kernel/docs/doxygen](https://github.com/kolibrios-nextgen/kolibrios-ng/tree/main/kernel/docs/doxygen)

Reply:
Added.